### PR TITLE
[12.0][IMP] sale_comment_template

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Forward port of  https://github.com/OCA/sale-reporting/pull/52

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
cc @MiquelRForgeFlow @ForgeFlow

**Review by rousseldenis — APPROVED** [review]
Code review

File: sale_comment_template/views/report_saleorder.xml [hunk 1]
```diff
@@ -3,11 +3,33 @@
 
     <template id="report_saleorder_document_comments"
       inherit_id="sale.report_saleorder_document">
+      <xpath expr="//div[@class='oe_structure']" position="before">
```
Comment (MiquelRForgeFlow) [inline sale_comment_template/views/report_saleorder.xml]:
```suggestion
      <xpath expr="//div[hasclass('oe_structure')]" position="before">
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
ready to merge

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**